<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryCell</class>
 <widget class="QWidget" name="InventoryCell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>68</width>
    <height>84</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="inventoryCellBorder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CentralElement">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inventoryCellNew">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="DropdownButton">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>56</y>
     <width>60</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inventoryCellBG">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>3</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Locked">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Blocked">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>66</width>
     <height>66</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Item" name="item">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ItemPixmapGrab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>CentralElement</zorder>
  <zorder>inventoryCellBG</zorder>
  <zorder>inventoryCellBorder</zorder>
  <zorder>Locked</zorder>
  <zorder>DropdownButton</zorder>
  <zorder>Blocked</zorder>
  <zorder>inventoryCellNew</zorder>
  <zorder>ItemPixmapGrab</zorder>
  <zorder>item</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Item</class>
   <extends>QPushButton</extends>
   <header location="global">CustomWidgets/Item/item.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
